Rakefile

```diff
@@ -4,6 +4,9 @@
 require 'bundler'
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+
+PROJECT_DIR = File.expand_path(".")
```

Comment:
Hmm, I wonder if there's a better way to do this then? I can probably get to this towards the end of the day and will think about it in the meantime.